components/availability/src/Availability.svelte

```diff
@@ -1305,73 +1385,109 @@
-                if (mouseIsDown) {
-                  endDrag(slot, day);
-                }
-              }}
-              on:keypress={(e) => {
-                if (e.code === "Space" || e.code === "Enter") {
+            {/each}
+          </div>
+          <div class="slots">
+            {#each day.slots as slot}
+              <button
+                data-available-calendars={slot.available_calendars.toString()}
+                aria-label="{new Date(
+                  slot.start_time,
+                ).toLocaleString()} to {new Date(
+                  slot.end_time,
+                ).toLocaleString()}}; Free calendars: {slot.available_calendars.toString()}"
+                class="slot {slot.selectionStatus} {slot.availability}"
+                class:pending={slot.selectionPending}
+                data-start-time={new Date(slot.start_time).toLocaleString()}
+                data-end-time={new Date(slot.end_time).toLocaleString()}
+                on:mousedown={() => {
+                  if (
+                    slotSelection.length < max_bookable_slots ||
+                    slot.selectionStatus === SelectionStatus.SELECTED
+                  ) {
+                    mouseIsDown = true;
+                  }
                   startDrag(slot, day);
-                  tick().then(() => endDrag(slot, day));
-                }
-              }}
-            >
-              {#if getBlockTimes(slot, day)}
-                <span class="selected-heading">{getBlockTimes(slot, day)}</span>
-              {/if}
-            </button>
-          {/each}
-        </div>
+                }}
+                on:mouseenter={() => {
+                  addToDrag(slot, day);
+                }}
+                on:mouseup={() => {
+                  if (mouseIsDown) {
+                    endDrag(slot, day);
+                  }
+                }}
+                on:keypress={(e) => {
+                  if (e.code === "Space" || e.code === "Enter") {
+                    startDrag(slot, day);
+                    tick().then(() => endDrag(slot, day));
+                  }
+                }}
+              >
+                {#if getBlockTimes(slot, day)}
+                  <span class="selected-heading"
+                    >{getBlockTimes(slot, day)}</span
+                  >
+                {/if}
+              </button>
+            {/each}
+          </div>
+        {:else if view_as === "list"}
+          <div class="slot-list">
+            {#each day.slots.filter((slot) => slot.availability === AvailabilityStatus.FREE || slot.availability === AvailabilityStatus.PARTIAL) as slot}
+              <button
+                data-available-calendars={slot.available_calendars.toString()}
+                aria-label="{new Date(
+                  slot.start_time,
+                ).toLocaleString()} to {new Date(
+                  slot.end_time,
+                ).toLocaleString()}; Free calendars: {slot.available_calendars.toString()}"
```

Comment:
Does this actually work? With the ""?
```suggestion
                aria-label={new Date(
                  slot.start_time,
                ).toLocaleString()} to {new Date(
                  slot.end_time,
                ).toLocaleString()}; Free calendars: {slot.available_calendars.toString()}
```

Reply:
Yep, and it's necessary as well here, because of the `...} to {...` bit!